src/routes/v2/organisation-schema.oas.yml

```diff
@@ -78,6 +78,20 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/BadRequest'
+        '500':
+          description: 'Internal server error.'
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  code:
+                    type: integer
+                    example: 500
+                  message:
+                    type: string
+                    example: >
+                      There was an error processing your request. It has been logged (ID 704b9f3b19536a35).'
```

Comment:
This should probably be extracted to a common response in the `components` section below, and then this and the other routes `$ref` it